examples/with-mongodb/README.md

```diff
@@ -0,0 +1,83 @@
+## Example app using MongoDB
+
+[MongoDB](https://mongodb.com/atlas) is a general purpose, document-based, distributed database built for modern application developers and for the cloud era. This example will show you how to connect to and use MongoDB as your backend for your Next.js app.
+
+- [MongoDB Atlas](https://mongodb.com/atlas)
+- [MongoDB Documentation](https://docs.mongodb.com/)
+
+## Demo
+
+## Deploy your own
+
+Once you have access to the environment variables you'll need, deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example):
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/import/git?c=1&s=https://github.com/vercel/next.js/tree/canary/examples/with-mongodb&env=MONGODB_URI,MONGODB_DB&envDescription=Required%20to%20connect%20the%20app%20with%20MongoDB)
+
+## How to use
+
+Using `create-next-app`
+
+Execute `create-next-app` with npm or Yarn to bootstrap the example:
+
+```bash
+npx create-next-app --example with-mongodb with-mongodb
+# or
+yarn create next-app --example with-mongodb with-mongodb
+```
+
+## Configuration
+
+### Set up a MongoDB database
+
+Set up a MongoDB database either locally or with [MongoDB Atlas for free](https://mongodb.com/atlas).
+
+### Set up environment variables
+
+Copy the `env.local.example` file in this directory to `.env.local` (which will be ignored by Git):
+
+```bash
+cp .env.local.example .env.local
+```
+
+Set each variable on `.env.local`:
+
+- `MONGODB_URI` - Your MongoDB connection string. If you are using [MongoDB Atlas](https://mongodb.com/atlas) you can find this by clicking the "Connect" button for your cluster.
+- `MONGODB_DB` - The name of the MongoDB database you want to use.
+
+### Start Up Development Server
+
+Navigate to your project directory and run either:
+
+```bash
+npm install
+npm run dev
+# or
+yarn
+yarn dev
+```
+
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+
+You will either see a message stating "You are connected to MongoDB" or "You are NOT connected to MongoDB". Ensure that you have provided the correct `MONGODB_URI` and `MONGODB_DB` environment variables.
+
+When you are successfully connected, you can refer to the [MongoDB Node.js Driver docs](https://mongodb.github.io/node-mongodb-native/3.4/tutorials/collections/) for further instructions on how to query your database.
+
+For a super quick example you can use the `db` variable that exported from the `mongodb.js` util file to insert data by running:
+
+```js
+db.collection('YOUR-COLLECTION-NAME').insertOne({ hello: 'world' })
+```
+
+Additionally, you can query the data by running:
+
+```js
+db.collection('YOUR-COLLECTION-NAME').find().toArray()
+```
+
+You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+
```

Comment:
This is not really related to the example but to documentation of the mongo driver, which is better featured in their official docs.

```suggestion
```